### ISSUE TITLE
Problem about the GCN results of Webnlg task

### ISSUE THREAD

**Issue by ghostyoona** [diegma/graph-2-text#9]
I'm interest in your paper, thank you for your work.
GCN result in Table 1 of the paper shows 53.5 bleu score.  But I can't get the score(highest score is 51.75) after running the project serval times(I have tried to change the batch_size and use gcn gates). 
My procedure lists as follow:
(1)Using preprocess.py to preprocess by train-delex files and dev-delex files, e.g train-webnlg-all-delex-src-nodes.txt.

(2)Then I run the train.py. python3 train.py -data data/gcn_exp -save_model data/tmp_ -rnn_size 256 -word_vec_size 256 -layers 1 -epochs 30 -optim adam -learning_rate 0.001 -encoder_type gcn -gcn_num_inputs 256 -gcn_num_units 256 -gcn_in_arcs -gcn_out_arcs -gcn_num_layers 4 -gcn_num_labels 5 -gcn-residual residual -seed 42(43 and 44)

(3)Using translate.py to generate text by test-delex files and relexicalise the text.

(4)Using calculate_bleu_dev.sh to evaluation relexicalised text with only three test-notdelex references(test-all-notdelex-reference0.lex, test-all-notdelex-reference1.lex, test-all-notdelex-reference2.lex).

Is my procedure right? Could you please provide me a exact hyparameters?

**Comment by crazyofapple** [comment]
Hello, I wonder if you can help me. I can't even get 10+ BLEU, is there something wrong? Here is my procedure @ghostyoona
`python3 train.py  -data data/gcn_exp -gcn_residual residual -save_model data/tmp_ -rnn_size 256 -word_vec_size 256 -layers 1 -epochs 30 -optim adam -learning_rate 0.001 -encoder_type gcn -gcn_num_inputs 256 -gcn_num_units 256 -gcn_in_arcs -gcn_out_arcs -gcn_num_layers 4 -gcn_num_labels 5  -seed 42


python translate.py -model data/tmp__acc_66.62_ppl_3.99_e6.pt -data_type gcn -src data/webnlg/dev-webnlg-all-delex-src-nodes.txt -tgt data/webnlg/dev-webnlg-all-delex-tgt.txt -src_label data/webnlg/dev-webnlg-all-delex-src-labels.txt -src_node1 data/webnlg/dev-webnlg-all-delex-src-node1.txt -src_node2 data/webnlg/dev-webnlg-all-delex-src-node2.txt -output data/webnlg/delexicalized_predictions_dev.txt -replace_unk -verbose


cd data/webnlg/
python3 ../../webnlg_eval_scripts/webnlg_gcnonmt_relexicalise.py -i ./ -f delexicalized_predictions_dev.txt
cd ../..
webnlg_eval_scripts/calculate_bleu_dev.sh
`
